test/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu
```

Comment:
You may want to use a tag other than the implicit `latest` for stability reasons here.